tests/storage/export.go

```diff
@@ -310,7 +310,7 @@ var _ = SIGDescribe("Export", func() {
 		Expect(md5sum).To(HaveLen(32))
 
 		err = virtClient.CoreV1().Pods(pod.Namespace).Delete(context.Background(), pod.Name, metav1.DeleteOptions{
-			GracePeriodSeconds: pointer.Int64(0),
+			GracePeriodSeconds: ptr.To[int64](0),
```

Comment:
Nice initiative to update the pointer but it is now recommended to use:
"kubevirt.io/kubevirt/pkg/pointer"
use by:
pointer.P(<anyTypeVariable>)
here for example:
pointer.P(0)

Reply:
Okay, I wish they would make up their minds...

Reply:
yeah for sure, I do hope this is the final pointer preferred repo